### PR TITLE
 Workflow for Github Actions to build and test OpenCV on Linux for 5.x

### PR DESCRIPTION
**Pull request by asenyaev**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ ] I agree to contribute to the project under Apache 2 License.
- [ ] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [ ] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: .github/workflows/PR-5.x-U20.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  PYTHONPATH: /opencv-contrib-build/python_loader:$PYTHONPATH
+
+jobs:
+  BuildAndTest:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: quay.io/asenyaev/opencv-ubuntu:20.04
```
Comment (fengyuentau) [inline .github/workflows/PR-5.x-U20.yaml:26]: `quay.io/asenyaev/opencv-ubuntu:20.04` -> `quay.io/opencv-ci/opencv-ubuntu:20.04`?

**Review by fengyuentau — APPROVED** [review]
👍

File: .github/workflows/PR-5.x-U20.yaml (hunk 1 above)
Comment (asenyaev) [reply .github/workflows/PR-5.x-U20.yaml:26]: Sure, it should be done in 3.4 for the reason to have a history about it in all branches merging one to another.

**Comment by asenyaev** [comment]
Closing this PR in case of it was added in a [dedicated repo](https://github.com/opencv/ci-gha-workflow/pull/2).

**Pull request closed without merging**